imports/plugins/included/products/productCategoryList.js

```diff
@@ -0,0 +1,61 @@
+
+export const productCategoryList =
+[ {
```

Comment:
Have you configured ESLINT for your editer, I guess such spacing can be flagged by the editor